container/sagemaker/tensorflow-serving.js

```diff
@@ -13,9 +13,44 @@ function invocations(r) {
     }
 }
 
-// ping check does not POST to a specific model, because MME starts with 0 model
 function ping(r) {
-    r.return(200, "OK!")
+    var uri = make_tfs_uri(r, false)
+
+    function callback (reply) {
+        if (reply.status == 200 && reply.responseBody.includes('"AVAILABLE"')) {
+            r.return(200)
+        } else {
+            r.error('failed ping' + reply.responseBody)
+            r.return(502)
+        }
+    }
+
+    r.subrequest(uri, callback)
+}
+
+function ping_tfs_1_11_or_mme(r) {
+    // hack for TF 1.11
```

Comment:
comment isn't correct anymore since we use this for more than just 1.11

Reply:
Updated the method comment.